osd_whats_new/osd-whats-new.adoc

```diff
@@ -122,7 +116,7 @@ For more information, see xref:../osd_planning/osd-limits-scalability.adoc#contr
 For more information about region availabilities, see xref:../osd_architecture/osd_policy/osd-service-definition.adoc#regions-availability-zones_osd-service-definition[Regions and availability zones].
```

Comment:
alignment here looks off, if you wanted to fix:
<img width="878" height="590" alt="image" src="https://github.com/user-attachments/assets/46f81d7e-5e4c-41e7-a240-ebd96214d93a" />

Reply:
Will fix when I moduralize the rest if the OSD whats new assembly. Thanks.